testing/asserts.ts

```diff
@@ -680,10 +680,10 @@ export function assertThrows<E extends Error = Error>(
 }
 
 /** Executes a function which returns a promise, expecting it to reject. */
-export function assertRejects(
+export function assertRejects<T = unknown>(
```

Comment:
Seems like an unrelated change, what's the reason for it?

Reply:
Ah, probably it's better to keep this unknown as the thrown value is not type checked

Reply:
Thanks for flagging, I will remove this change and refactor the test to use the format outlined below